catalog/core/catalog-core-commons/src/main/java/ddf/catalog/filter/delegate/TagsFilterDelegate.java

```diff
@@ -62,7 +62,7 @@ public Boolean or(List<Boolean> operands) {
 
   @Override
   public Boolean not(Boolean operand) {
-    return tags == null ? operand : !operand;
```

Comment:
Are we sure about changing this? If I remember it was to fix an issue with not respecting `not` correctly.

Reply:
Yes. The TagsFilterDelegate is just trying to determine if each clause has the needed metacard-tag property. In the case of the not clause we just want to pass along the result of whatever was in it.